backend/tracim_backend/migration/versions/ae30497b6168_taking_care_of_moved_contents_properties.py

```diff
@@ -0,0 +1,50 @@
+"""taking care of moved contents properties
+
+Revision ID: ae30497b6168
+Revises: 78a01733957f
+Create Date: 2021-03-16 15:59:36.034739
+
+"""
+import json
+
+from alembic import op
+import sqlalchemy as sa
+
+from tracim_backend.models.meta import metadata
+
+# revision identifiers, used by Alembic.
+revision = "ae30497b6168"
+down_revision = "78a01733957f"
```

Comment:
Depending which one of this or https://github.com/tracim/tracim/pull/4278 is merged first, we do need to adapt the down revision in one of those two PR.

Reply:
Yes, I thought about it.